packages/material-ui-lab/src/Slider/Slider.js

```diff
@@ -0,0 +1,549 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import withStyles from '../../../material-ui/src/styles/withStyles';
+import ButtonBase from '../../../material-ui/src/ButtonBase';
+import { fade } from '../../../material-ui/src/styles/colorManipulator';
```

Comment:
These should be imported as:

```
import withStyles from 'material-ui/styles/withStyles';
```
etc.

Thought we fixed this already, but hey! 😄